openhands/server/listen.py

```diff
@@ -55,14 +57,18 @@
 from openhands.server.auth import get_sid_from_token, sign_token
 from openhands.server.session import SessionManager
 
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
why is this here?

Reply:
For the SaaS mode GitHub OAuth env variables. The last endpoint in `openhands/server/listen.py` (`'/github/callback'`) needs them to retrieve the `access_token`